test/functionalTest/cases/2193_v1_reply_to_v2_response_on_error_with_cpr/forward_using_GET_v2_entities.test

```diff
@@ -0,0 +1,220 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Those tests in 0787 were pure copies of oldertests in 0787, converted to v2.
These new v2 tests have no v1 counterpart and don't belong in 0787.
Perhaps we should move those v2 tests from 0787 to 2193?